pysatNASA/instruments/timed_guvi.py

```diff
@@ -0,0 +1,298 @@
+Analysis Web (CDAWeb).
+
+Supports two options for loading that may be
+specified at instantiation.
+
+Properties
+----------
+platform
+    'timed'
+name
+    'guvi'
+tag
+    None
+inst_id
+    None supported
+
+Note
+----
+- no tag required
+
+Warnings
+--------
+- Currently no cleaning routine.
+
+
+Example
+-------
+::
+
+    import pysat
+    guvi = pysat.Instrument(platform='timed', name='guvi')
+    guvi.download(dt.datetime(2005, 6, 20), dt.datetime(2005, 6, 21))
+    guvi.load(yr=2005,doy=171)
+
+Author
+------
+L. A. Navarro (luis.navarro@colorado.edu)
+
+"""
+
+import datetime as dt
+import functools
+import pandas as pds
+import xarray as xr
+import numpy as np
+import warnings
+import pysat
+from datetime import datetime, timedelta
+
+from pysat import logger
+from pysat.instruments.methods import general as mm_gen
+
+from pysatNASA.instruments.methods import cdaweb as cdw
+from h5py._hl import dataset
+
+# ----------------------------------------------------------------------------
+# Instrument attributes
+
+platform = 'timed'
+name = 'guvi'
+tags = {'': ''}
+inst_ids = {'': [tag for tag in tags.keys()]}
+
+# ----------------------------------------------------------------------------
+# Instrument test attributes
+
+_test_dates = {'': {'': dt.datetime(2005, 6, 20)}}
+
+# ----------------------------------------------------------------------------
+# Instrument methods
+
+
+def init(self):
+    """Initializes the Instrument object with instrument specific values.
+
+    Runs once upon instantiation.
+
+    """
+
+    rules_url = 'http://guvitimed.jhuapl.edu/home_guvi-datausage'
+    ackn_str = ' '.join(('Please see the Rules of the Road at', rules_url))
+    logger.info(ackn_str)
+    self.acknowledgements = ackn_str
+    self.references = ' '.join(('Paxton,L. J., Christensen, A. B., Humm,',
+                                'D. C., Ogorzalek, B. S., Pardoe, C. T.,',
+                                'Monison, D., Weiss, M. B., Cram, W.,',
+                                'Lew, P. H., Mabry, D. J., Goldstena,',
+                                'J. O., Gary, A., Persons, D. F., Harold,',
+                                'M. J., Alvarez, E. B., ErcoF, C. J.,',
+                                'Strickland, D. J., Meng, C.-I.',
+                                'Global ultraviolet imager (GUVI): Measuring',
+                                'composition and energy inputs for the NASA',
+                                'Thermosphere Ionosphere Mesosphere Energetics',
+                                'and Dynamics (TIMED) mission.',
+                                'Optical spectroscopic techniques and',
+                                'instrumentation for atmospheric and',
+                                'space research III. Vol. 3756.',
+                                'International Society for Optics',
+                                'and Photonics, 1999.'))
+    self.pandas_format = False
```

Comment:
Recommend moving the setting of `pandas_format` up to the "Instrument attribute" block.